### PR TITLE
Add scout gem for memory leak detection

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**
**While [Scout is Beta on the Heroku](https://elements.heroku.com/addons/scout) it is free.**

Scout helps to determining memory leak on `Controller#action` basis:

medium

Also `scout` can determine `N+1` queries, but this feature is not main in our case.

### CODE REVIEW

**Comment by ruslankhaertdinov** [comment]
@timurvafin please take a look.

File: Gemfile [hunk 1]
```diff
@@ -45,6 +45,7 @@ gem "stackprof"
 group :staging, :production do
   gem "newrelic_rpm"
   gem "rails_stdout_logging"
+  gem "scout_apm", "~> 2.0.x"
```
Comment (timurvafin) [inline Gemfile]: What is the reason to use `"~> 2.0.x"` here?

Comment (ruslankhaertdinov) [inline Gemfile]: I copy-past it from documentation, but now can't find from where. I'll remove it.

**Comment by ruslankhaertdinov** [comment]
@timurvafin  I think we should use this service only if memory leaks occurs, otherwise it may cause performance issues (because it scans each request and method executions). If so we can close this PR.

**Comment by timurvafin** [comment]
@ruslankhaertdinov so you are ok to close this PR without merging?

**Comment by ruslankhaertdinov** [comment]
@timurvafin yes

**Comment by timurvafin** [comment]
@ruslankhaertdinov thanks, closed.

**Pull request closed without merging**